pyvista/core/objects.py

```diff
@@ -380,3 +380,25 @@ def get_data_range(  # type: ignore[override]
             return (np.nan, np.nan)
         # Use the array range
         return np.nanmin(arr), np.nanmax(arr)
+
+    @property
+    def is_empty(self) -> bool:  # numpydoc ignore=RT01
+        """Return ``True`` if the table has no rows or columns.
```

Comment:
is it possible to have rows but no columns or vice versa?  The text here has `or` but the conditional in the property has `and`.